VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -171,19 +172,9 @@
                         tools:targetApi="17" />
 
                     <Switch
+                        style="@style/edX.Widget.Switch"
```

Comment:
`android:switchStyle` is not available until API 21, and doesn't appear to be back-ported by appcompat, so we're stuck setting it on each individual switch for now.